contrib/gitian/README.md

```diff
@@ -76,15 +80,16 @@ This setup is required to enable networking in the container.
 Docker
 ------
 
-Building in docker does not require much setup. Install docker on your host, then type the following:
+Prepare for building with docker:
 
 ```bash
-sudo apt-get install git make curl
+sudo apt-get install git make curl docker.io
+sudo groupadd docker
```

Comment:
We should warn against this step, since it basically gives root access to docker: https://docs.docker.com/v17.09/engine/installation/linux/linux-postinstall/ . When building from a vm, this should be fine though.

Reply:
Okay, added a note and the link to explain security concerns.